rtrlib/lib/ordered_dyn_array.h

```diff
@@ -0,0 +1,96 @@
+/*
+* This file is part of RTRlib.
+*
+* This file is subject to the terms and conditions of the MIT license.
+* See the file LICENSE in the top level directory for more details.
+*
+* Website: http://rtrlib.realmv6.org/
+*/
+
+#ifndef LRTR_VECTOR_UTILS_H
+#define LRTR_VECTOR_UTILS_H
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+/**
+ * @brief ASPA Object
+ * here the customer as number (CAS) authorizes a set of provider AS numbers
+ */
+struct aspas_record {
+	uint32_t customer_asn;
+	size_t provider_count;
+	uint32_t *provider_asns;
+};
+
+/**
+ * @brief Struct which is similar in function to std::vector from C++.
```

Comment:
Instead of referencing std::vector here, it would be better to describe what it does.
Otherwise the reader first have to figure out how std::vector functions to know what ordered_dyn_array does